weaviate/templates/weaviateServiceGRPC.yaml

```diff
@@ -6,6 +6,9 @@ metadata:
   labels:
     app.kubernetes.io/name: weaviate
     app.kubernetes.io/managed-by: helm
+{{- with .Values.service.labels }}
+{{ toYaml . | indent 4 }}
+{{- end }}
```

Comment:
Can you add tests for this? You can use [check_string_existence](https://github.com/weaviate/weaviate-helm/blob/master/.cicd/test.sh#L154) method for that